lib/kvdb/kvdb_pfxlock.c

```diff
@@ -63,8 +63,8 @@ struct kvdb_pfxlock_tree {
     struct rb_root             kplt_root HSE_L1X_ALIGNED;
     struct kvdb_pfxlock_entry *kplt_ecache;
     volatile uint              kplt_entry_cnt;
-    uint                       kplt_mcache_cnt;
-    struct kvdb_pfxlock_entry *kplt_mcache;
+    uint                       kplt_cache_cnt;
+    struct kvdb_pfxlock_entry *kplt_cache;
```

Comment:
This got caught up in the rename even though it's not "mcache".  The name "cache" seemed as appropriate as "mcache", and less confusing in the short term.  I saw no clear description in the prefix lock code of what the "m" stands for.